commit-graph.c

```diff
@@ -24,7 +24,9 @@
 #define GRAPH_CHUNKID_DATA 0x43444154 /* "CDAT" */
 #define GRAPH_CHUNKID_EXTRAEDGES 0x45444745 /* "EDGE" */
 #define GRAPH_CHUNKID_BASE 0x42415345 /* "BASE" */
-#define MAX_NUM_CHUNKS 5
+#define GRAPH_CHUNKID_BLOOMINDEXES 0x42494458 /* "BIDX" */
```

Comment:
This commit should have the change to `test-tool read-graph` to include these chunk ids.

(Again, mention that you plan to split the bloom test script across multiple commits, but include it in one place to make your big changes easier.)